Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +9,66 @@ class Program
     {
         static void Main(string[] args)
         {
+            // Need to refactor the code I made in Lesson 2.
+            // Use functions to achieve DRY (as little duplicate code as possible).
+
+            // Name input
+            string name = PromptString("Enter your name: ");
+
+            // Surname input
+            string surname = PromptString("Enter your surname: ");
+
```

Comment:
I would remove a blank line between the reads, they are related and can be sticked right next to each other. For bmi calculation- either way is fine.

Reply:
Done, blank lines removed.